Nos.xcodeproj/xcshareddata/xcschemes/Nos.xcscheme

```diff
@@ -111,6 +111,12 @@
             ReferencedContainer = "container:Nos.xcodeproj">
          </BuildableReference>
       </BuildableProductRunnable>
+      <CommandLineArguments>
```

Comment:
This should be reverted before we merge as it slows down the queries quite a bit.